### DISCUSSION TITLE
Plugins not starting

### DISCUSSION THREAD

**Discussion by harunnoir** [lumen-oss/rocks.nvim#332] [Q&A] [answered]
I installed this packages, but they do not start.

![image](https://github.com/nvim-neorocks/rocks.nvim/assets/152083466/8ed81142-1d95-4c67-afae-778ecc380a1a)

When I check: Rocks log I get:

```
[START][2024-05-15 06:58:19] rocks.nvim logging initiated
ERROR | 2024-05-15 06:58:19 | .../.local/share/nvim/rocks/share/lua/5.1/rocks/adapter.lua:53 | Failed to remove symlink: /home/x_user/.local/share/nvim/rocks/rocks_rtp/parser (ENOENT: no such file or directory: /home/user/.local/share/nvim/rocks/rocks_rtp/parser)
```

please how to solve the problem

**Comment by mrcjkb** [comment, marked as answer]
Hey 👋 

That error log isn't what's preventing the startup. 
Some lua plugins require you to call a `setup` function to initialise them.

**Comment by harunnoir** [comment]
> Hey 👋
> 
> That error log isn't what's preventing the startup. Some lua plugins require you to call a `setup` function to initialise them.

Today is my first day with rocks and I actually do not know how to deal with it.

**Comment by harunnoir** [comment]
> Hey 👋
> 
> That error log isn't what's preventing the startup. Some lua plugins require you to call a `setup` function to initialise them.

But in the readme section there's

![image](https://github.com/nvim-neorocks/rocks.nvim/assets/152083466/85bd820b-144b-4565-83c2-e15348699522)

**Reply by mrcjkb** [reply]
Yes, but that is useless if the plugin doesn't automatically initialise itself (i.e. doesn't provide anything to source).

This is a problem [I've written about in detail here](https://mrcjkb.dev/posts/2023-08-22-setup.html).

If you want to work around this, you can install [rocks-config.nvim](https://github.com/nvim-neorocks/rocks-config.nvim) and set `auto_setup = true`. Just make sure to read the [warning box](https://github.com/nvim-neorocks/rocks-config.nvim?tab=readme-ov-file#auto_setup).

**Reply by harunnoir** [reply]
Thanks a lot!

**Reply by mrcjkb** [reply]
No problem.
Let me know if it works out for you. 😄

**Reply by harunnoir** [reply]
I totally like it, it is really good...
